e2e/tests/stablex_test.rs

```diff
@@ -120,7 +120,7 @@ fn test_with_ganache() {
     assert_eq!(balance_after - balance_before, 999_000.into())
 }
 
-ethcontract::contract!("dex-contracts/build/contracts/BatchExchange.json");
+ethcontract::contract!("dex-contracts/build/contracts/BatchExchange.json"); // Isn't this already available in lib.rs?
```

Comment:
@fleupold - I think this was left over from a time when this file structure was different (and could be removed).

Reply:
👍

Reply:
I wasn't able to remove this without the compiler complaining that `BatchExchange` does not exist.

Reply:
You probably have to `use crate::BatchExchange` or something similar.

Reply:
is this a "crate"?

Reply:
`e2e` is a crate (a Rust word for library), its where the testing utilities are. `crate` is a special keyword that means the local crate that the code is part of.

My initial suggesting was wrong, as I thought that `BatchExchange` was in the same crate as this line of code, but that is not the case.